src/debug/construct.js

```diff
@@ -0,0 +1,20 @@
+'use babel'
+
+import debugInfo from './info'
+
+const constructDebug = async () => {
```

Comment:
Debug _what_? `Object`? `Array`? (There's a reason the original function was `generateDebugString`)

Reply:
I don't typically expect the datatype of a return value to be part of a function name, unless it distinguishes it from a similar function with a different return type.

Is there alternatively a debug object or a debug array that I missed? When we want "A Debug" to provide to the user, we call this function. The `debugInfo` submodule returns and object, and you aren't requesting to rename that to `debugInfoObject`.

I think a better solution here is to get this `flow` thing in place and define a return type.

Reply:
I think having `string` in the name was more important when this was all flat files. Much easier to get it mixed up with the `getDebugInfo`, which was in the same file and in the same scope and returns an object. Now, `debug.construct` has a clearly hierarchically superior position over its submodule.

Reply:
Hmm. Leaving the old arguments up partly for posterity and partly because they still make some useful points. But I'm looking closer at the code and starting to rethink this a bit.

The object returned from `debug.info` really is everything that makes up a debug, except for some labeling for string output. Perhaps, `debug.userMessage`? or something like that. I guess the target is us, more-so than the user, so maybe not quite right. But namely I'm _much_ more interested in a name that captures its intent of prepping it for display, than describing it as a `string`.

Reply:
`debugMessage`?

I like the old name because it says _exactly_ what it is doing, when I saw `constructDebug` I had no idea what it was doing until I looked at the code and saw what it used to be.

`debug.label` doesn't work since this isn't just a label but the entire message.